### DISCUSSION TITLE
Use Arrow-Keys to browse Command-History

### DISCUSSION THREAD

**Discussion by apettel** [vscode-neovim/vscode-neovim#1725] [Q&A]
I would like to use the up/down arrow keys to scroll through the command-history in command mode. I tried to remap \<Up> to \<C-p> and \<Down> to \<C-n> without success.

**Comment by xiyaowong** [comment]
Use the vscode-neovim.send and vscode-neovim.send-cmdline commands to send keys to nvim

**Comment by theol0403** [comment]
Not possible

https://github.com/vscode-neovim/vscode-neovim/issues/288
